build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+dotnet build --configuration Release
+rm -rf ccgateway/bin/Release/netcoreapp3.1/plugins
+mkdir ccgateway/bin/Release/netcoreapp3.1/plugins
+cp ccgateway/plugins/*.dll ccgateway/bin/Release/netcoreapp3.1/plugins/
```

Comment:
I'd prefer these steps to be inside of the Dockerfile so that no matter on what system we execute this script we get the same results and everything is self-contained inside the container? What do you think about that ?

Note: maybe this is opening up a bigger discussion than originally intended by the Jira ticket.

Reply:
Looks good,
Just wondering if we should migrate the output to another directory. We should be copying the framework out to a running destination like. /opt/ccgateway.

Reply:
@atodorov Moved the steps directly to the dockerfile